db/seed/example_studies/exp_cell_filtering/study_info.json

```
@@ -0,0 +1,36 @@
+{
+  "study": {
+    "name": "Expression Matrix Cell Filtering",
+    "description": "<p>Example study showing effects of filtering distribution-based plots for cells only present in processed expression matrices.</p>",
```

Enclosing this in `p` tags seems odd, but I imagine it's merely a vestige of previous synthetic studies.

```suggestion
    "description": "<p>Example study showing effects of filtering distribution-based plots for cells only present in processed expression matrices.</p>",
```

---

Agreed, that's a copy/paste.